modules/openapi-generator/src/main/resources/csharp-netcore/Configuration.mustache

```diff
@@ -223,6 +225,13 @@ namespace {{packageName}}.Client
         /// <value>The access token.</value>
         public virtual string AccessToken { get; set; }
 
+
+        /// <inheritdoc />
+        public JsonSerializerSettings SerializerSettings { get; set; } = new JsonSerializerSettings();
```

Comment:
I think this will need to be moved elsewhere… maybe as a constructor argument for `ApiClient`.

`Configuration` should be decoupled from framework implementation. For example, `ApiClient` is tightly coupled to RestSharp and JSON.net.  Having `Configuration` rely only on "standard" types means that users may replace `ApiClient` with their desired accessor implementations. Including Json.NET types in Configuration will make this unusable for users who don't pull that type into their client.

Reply:
Its possible to move it to the constructor. But everything is configurated inside the `Configuration `type but just not this? I think every C# developer use `Newtonsoft.Json`. Even if the `ApiClient ` changes. When it moved to the constructor of the `ApiClient ` , the class that wrap this must also use `Newtonsoft.Json`. 
If you want to use this type just inside the `ApiClient `, we have to create a custom type and map everything. But this is not a good way.